server/engine/src/main/resources/META-INF/native-image/resource-config.json

```diff
@@ -0,0 +1,51 @@
+{
+  "resources":{
+    "includes":[
+      {
+        "pattern":"\\QimplicitCopybooks/DFHEIBLC.cpy\\E"
```

Comment:
I'm not sure I understand what are those?

Reply:
Graal native image compiles bytecode Ahead-Of-Time, and create a native image. But it is not aware of certain things ahead of time like proxy object/reflection/resource files we need. So, we create these configuration for graalVm to know and include it. There is a utility from graalVm to generate these config files. By default graal uses configs directly from META-INF/native-image folder.

Reply:
👍